src/tribler/core/components/libtorrent/download_manager/download_manager.py

```diff
@@ -374,19 +400,19 @@ def get_upload_rate_limit(self, hops=None):
         # Rate conversion due to the fact that we had a different system with Swift
         # and the old python BitTorrent core: unlimited == 0, stop == -1, else rate in kbytes
         libtorrent_rate = self.get_session(hops).upload_rate_limit()
-        return 0 if libtorrent_rate == -1 else (-1 if libtorrent_rate == 1 else libtorrent_rate / 1024)
+        return self.__class__.reverse_convert_rate(rate=libtorrent_rate)
```

Comment:
This version is a bit easier to read:

```suggestion
        return self.reverse_convert_rate(rate=libtorrent_rate)
```